README.md

```diff
@@ -1,69 +1,149 @@
-<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Inc. logo" title="Google" align="right" height="96" width="96"/>
+[//]: # "This README.md file is auto-generated, all changes to this file will be lost."
+[//]: # "To regenerate it, use `python -m synthtool`."
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
 
-# Google Cloud Firestore Sessions
+# [Google Cloud Firestore Session: Node.js Client](https://github.com/googleapis/nodejs-firestore-session)
```

Comment:
@bcoe this library doesn't fit the traditional mold, in that it's a connector sorta thing.  The title is sorta goofy - any ideas how we can better support this in the partials template?

Reply:
Good question, do you think we should just let you override the title; make it optional in the partials?

Any other thoughts?